cmd/envtool/envtool.go

```diff
@@ -208,11 +209,76 @@ func runCommand(command string, args []string, stdout io.Writer, logger *zap.Sug
 	cmd := exec.Command(bin, args...)
 	logger.Debugf("Running %s", strings.Join(cmd.Args, " "))
 
-	cmd.Stdout = stdout
 	cmd.Stderr = os.Stderr
+	p, err := cmd.StdoutPipe()
```

Comment:
I think we should create a separate function to run tests instead of adding a hack to general-purpose `runCommand` function